### PR TITLE
allow the voice changing

### PR DESCRIPTION
**Pull request by nataliadiak**

### CODE REVIEW

**Review by met — APPROVED** [review]
Look good.

I try to merge, and run it with updated data with some russian voice.

**Comment by met** [comment]
Přidávám odkaz na Trello s detailnějším popisem co a proč tahle změna dělá:
https://trello.com/c/xbHd05tN/427-umo%C5%BEnit-ssml-v-airtable-m%C4%9Bnit-hlas

File: src/translationPipes/GenerateSound.ts [hunk 1]
```diff
@@ -137,16 +137,22 @@ export class GenerateSound implements TranslationPipe {
                 audioConfig
             )
 
+            const childElement = /^<voice/.test(text)
```
Comment (pionl) [inline src/translationPipes/GenerateSound.ts:140]: I think next time [startsWith function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) would be better here (few nano seconds faster? :D ).

Comment (met) [reply src/translationPipes/GenerateSound.ts:140]: Good idea @nataliadiak please, can you make this small change? (New PR)

Comment (nataliadiak) [reply src/translationPipes/GenerateSound.ts:140]: Sure.

Comment (pionl) [reply src/translationPipes/GenerateSound.ts:140]: I think we can leave it for future changes, it works same at this moment :)

But. We should probably do a text.trim() first if there is any accidental whitespace

**Pull request merged**